pkg/util/k8sutil/k8sutil.go

```diff
@@ -248,6 +252,22 @@ func NewZookeeperPod(m *zookeeperutil.Member, existingCluster []string, clusterN
 		zooServers[len(existingCluster)] = fmt.Sprintf("server.%d=%s:2888:3888:observer;%s:2181", m.ID(), m.Addr(), m.Addr())
 	}
 
+	cpus, err := resource.ParseQuantity(cs.RequestCPU)
+	if err != nil {
+		cpus, _ = resource.ParseQuantity(DefaultCPU)
+	}
+
+	memory, err := resource.ParseQuantity(cs.RequestMEM)
+	if err != nil {
```

Comment:
can we inform the user that the requested Memory is not parsable? Silently applying a default setting might be hard to debug.